### PR TITLE
Change Text from ECS to EKS in Central Dashboard

### PR DESCRIPTION
**Pull request by jsitu777**
**Which issue is resolved by this Pull Request:**
Resolves #

**Description of your changes:**
- make kustomize overlay to change text from ECS to EKS in Central-Dashboard

**Testing:**
- [ ] Unit tests pass
- [ ] e2e tests pass
- Details about new tests (If this PR adds a new feature)
- Details about any manual tests performed

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: website/content/en/docs/deployment/cognito/manifest/guide.md [hunk 1]
```diff
@@ -145,7 +145,7 @@ make deploy-kubeflow INSTALLATION_OPTION=helm DEPLOYMENT_OPTION=cognito
         kustomize build upstream/apps/katib/upstream/installs/katib-with-kubeflow | kubectl apply -f -
         
         # Central Dashboard
-        kustomize build upstream/apps/centraldashboard/upstream/overlays/kserve | kubectl apply -f -
+        kustomize build awsconfigs/apps/centraldashboard | kubectl apply -f -
```
Comment (surajkota) [inline website/content/en/docs/deployment/cognito/manifest/guide.md:148]: remove the option to install individual components

**Comment by ryansteakley** [comment]
Looks like we should open a pr for this https://github.com/kubeflow/kubeflow/blob/master/components/centraldashboard/config/centraldashboard-config.yaml edit at some point rather than doing this we might miss changes in the dashboard. For now I guess this is fine not sure if its possible to patch this instead of copying over the entire file?

File: awsconfigs/apps/centraldashboard/patches/configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+            "desc": "A fast and easy way to deploy Kubeflow locally",
+            "link": "https://www.kubeflow.org/docs/distributions/minikf/"
+          },
+          {
+            "text": "Microk8s for Kubeflow",
+            "desc": "Quickly get Kubeflow running locally on native hypervisors",
+            "link": "https://www.kubeflow.org/docs/distributions/microk8s/kubeflow-on-microk8s/"
+          },
+          {
+            "text": "Kubeflow on GCP",
```
Comment (surajkota) [inline awsconfigs/apps/centraldashboard/patches/configmap.yaml:117]: remove Kubeflow on GCP, Microk8s for Kubeflow, MiniKF

and add
```
{
            "text": "Amazon SageMaker Components for Kubeflow Pipelines",
            "desc": "Create Machine Learning workflows to run on Amazon SageMaker from Kubeflow Pipelines",
            "link": "https://awslabs.github.io/kubeflow-manifests/docs/amazon-sagemaker-integration/sagemaker-components-for-kubeflow-pipelines"
},
{
            "text": "Amazon SageMaker Operator for Kubernetes (ACK)",
            "desc": "Use SageMaker Operators for Kubernetes (ACK) to manage your SageMaker resources from your Kubernetes cluster directly",
            "link": "https://awslabs.github.io/kubeflow-manifests/docs/amazon-sagemaker-integration/sagemaker-operators-ack"
},
```

**Review by surajkota — CHANGES_REQUESTED** [review]

File: awsconfigs/apps/centraldashboard/kustomization.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../../../upstream/apps/centraldashboard/upstream/overlays/istio
```
Comment (surajkota) [inline awsconfigs/apps/centraldashboard/kustomization.yaml:5]: wrong location, this should be

apps/centraldashboard/upstream/overlays/kserve. see https://github.com/awslabs/kubeflow-manifests/blob/main/deployments/vanilla/kustomization.yaml#L32

do we need to regenerate the charts?

Comment (jsitu777) [reply awsconfigs/apps/centraldashboard/kustomization.yaml:5]: this is the right location.  resource here is point to istio https://github.com/kubeflow/manifests/blob/master/apps/centraldashboard/upstream/overlays/kserve/kustomization.yaml

**Review by surajkota — APPROVED** [review]

**Pull request merged**